src/custom-elements-es5-adapter.ts

```diff
@@ -31,22 +31,10 @@ const styleMatcher = p.AND(
     p.hasTagName('style'),
     p.OR(p.NOT(p.hasAttr('type')), p.hasAttrValue('type', 'text/css')));
 
-
 /**
- * When compiling ES6 classes down to ES5 we need to include a special form of
- * the webcomponents loader to be compatible with native custom elements. This
- * includes moving the loader script element and all relevent siblings
- * following it into the body so that the Custom Elements ES5 adapter can be
- * properly injected.
- *
- * NOTE(fks) 03-28-2017: Comments are not currently moved to the body, which
- * means relevent comments may be left behind in the head. This is fine for now,
- * but may break if we ever decide to support important metadata in comments
- * (ex: support for defining lazy-imports via HTML comments).
- *
- * TODO(fks) 03-28-2017: Move into polymer-build and add tests.
+ * Wraps `addCustomElementsEs5Adapter()` in a `stream.Transform`.
  */
-export class InjectES5Adapter extends stream.Transform {
+export class CustomElementsEs5AdapterTransformer extends stream.Transform {
```

Comment:
such a great bikeshedding opportunity: nice noun phrase :) but we don't need to append the type, we can just use the description of what it does, like `CustomElementsEs5AdapterInjector`